helm/alloy/templates/vertical-pod-autoscaler.yaml

```diff
@@ -0,0 +1,42 @@
+### Upstream work being tracked at https://github.com/grafana/alloy/pull/1305
+{{ if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" -}}
```

Comment:
I think we don't even need this as Alloy will only be deployed on clusters using recent releases (as of now) which all have the VPA CRDs embedded.

Reply:
True but this will be upstream anyway :D

Reply:
Oh ok nevermind then :)